src/app/admin/deleteUser/DeleteUserDialog.tsx

```diff
@@ -36,29 +38,59 @@ const DeleteUserDialog: React.FC<Props> = (props) => {
             return;
         }
 
-        const { error } = await adminDeleteUser(props.userToDelete.id);
+        const { data: profileUserId, error: profileUserIdError } = await supabase
```

Comment:
I wonder if we should get the profile id and send the audit log within `adminDeleteUser` instead?